protocol/xmrmaker/swap_state_test.go

```diff
@@ -100,7 +100,7 @@ func newTestXMRMaker(t *testing.T) *Instance {
 func newTestInstance(t *testing.T) (*Instance, *swapState) {
 	xmrmaker := newTestXMRMaker(t)
 	infoFile := path.Join(t.TempDir(), "test.keys")
-	swapState, err := newSwapState(xmrmaker.backend, &types.Offer{}, xmrmaker.offerManager, nil, infoFile,
+	swapState, err := newSwapState(xmrmaker.backend, types.NewOffer("", 0, 0, 0), xmrmaker.offerManager, nil, infoFile,
```

Comment:
I was just keeping the test the way it was, but it does seems a little strange that `newSwapState` accepts that offer as valid input.

Reply:
yeah, it wouldn't be bad to validate that the offer min/max amounts and exchange rate are nonzero